persist-cli-tests/src/test/resources/test-src/input/tool_test_generate_17/main.bal

```diff
@@ -0,0 +1,15 @@
+import ballerina/io;
```

Comment:
add license headers

Reply:
Shall we use LLC format

Reply:
updated in [682be05](https://github.com/ballerina-platform/persist-tools/pull/46/commits/682be05657faaa87c11fcd4c856559e1b84b2367)